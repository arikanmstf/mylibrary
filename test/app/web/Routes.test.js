import React from 'react';
import { shallow } from 'enzyme';

describe('test/app/web/Routes.test.js', () => {
  let Routes;

  beforeAll(() => {
    jest.mock('assets/images/loading.gif', () => 'mockLoading');
    Routes = require('web/Routes').default; // eslint-disable-line
  });

  it('Render: props: null', () => {
    const wrapper = shallow(<Routes />);
    expect(wrapper).toMatchSnapshot();
  });
});
