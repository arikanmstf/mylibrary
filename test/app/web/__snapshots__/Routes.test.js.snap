// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test/app/web/Routes.test.js Render: props: null 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "HomeScreen": [Function],
      "Loader": [Function],
      "LoginScreen": [Function],
      "PrivateRoute": [Function],
      "Route": [Function],
      "Router": [Function],
      "Row": [Function],
      "Switch": [Function],
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "HomeScreen": [Function],
        "Loader": [Function],
        "LoginScreen": [Function],
        "PrivateRoute": [Function],
        "Route": [Function],
        "Router": [Function],
        "Row": [Function],
        "Switch": [Function],
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
